sys/winsize.go

```diff
@@ -1,22 +1,30 @@
-package sys
+// Copyright 2015 go-termios Author. All Rights Reserved.
+// https://github.com/go-termios/termios
+// Author: John Lenton <chipaca@github.com>
 
-/*
-#include <termios.h>
-#include <sys/ioctl.h>
+package sys
 
-void getwinsize(int fd, int *row, int *col) {
-	struct winsize wsz;
-	ioctl(fd, TIOCGWINSZ, &wsz);
-	*row = wsz.ws_row;
-	*col = wsz.ws_col;
-}
-*/
-import "C"
+import (
+	"fmt"
+	"golang.org/x/sys/unix"
+	"unsafe"
+)
 
 // GetWinsize queries the size of the terminal referenced by the given file
 // descriptor.
+//
+type winSize struct {
```

Comment:
Please move this struct above GetWinsize's docstring.

Also, this is one of the few places where we have to rely on the size of the fields. Please include a comment like this:

// winSize mirrors struct winsize in the C header. The following declaration matches struct winsize in the headers of Linux and FreeBSD.

(I've verified that.)